src/flint/fmpz_mpoly.jl

```diff
@@ -636,6 +636,33 @@ function neg!(z::ZZMPolyRingElem, a::ZZMPolyRingElem)
   return z
 end
 
+#
+
+function set!(z::ZZMPolyRingElem, a::ZZMPolyRingElem)
```

Comment:
Maybe should also do this to make it more generally useful (based on a concrete need I just had ;-) )
```suggestion
function set!(z::ZZMPolyRingElemOrPtr, a::ZZMPolyRingElemOrPtr; parent::ZZMPolyRing=parent(a))
  @ccall libflint.fmpz_mpoly_set(z::Ref{ZZMPolyRingElem}, a::Ref{ZZMPolyRingElem}, parent::Ref{ZZMPolyRing})::Nothing
```

Or maybe the kwarg should be `ctx` to better fit with that @lgoettgens does in PR #1911 (which I kinda forgot about, I thought this was long merged... perhaps we can get together on gather later or next week to discuss this)